src/index.js

```diff
@@ -118,9 +118,13 @@ async function main({
         body: e.message,
       };
     }
-    log.error('Unhandled error', e, e.stack);
+    log.error('Unhandled error', e,
+    /* istanbul ignore next */
+      e.stack || 'no stack');
```

Comment:
but this doesn't prevent this error:

```
Cannot read property 'stack' of undefined
```

should be:

```
(e && e.stack) || 'no stack'
```

Reply:
```suggestion
      (e && e.stack) || 'no stack');
```